src/js/components/search/filters/awardType/PrimaryAwardType.jsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * primaryAwardType.jsx
+ * Created by Emily Gullo 11/02/2016
+ **/
+
+import React from 'react';
+import * as Icons from '../../../sharedComponents/icons/Icons';
+import SecondaryAwardType from './SecondaryAwardType';
+
+const propTypes = {
+    name: React.PropTypes.string,
+    subList: React.PropTypes.array
+};
+
+export default class PrimaryAwardType extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            showSubItems: false
+        };
+    }
+
+    toggleSubItems() {
+        if (this.state.showSubItems === false) {
+            this.setState({ showSubItems: true });
+        }
+        else {
+            this.setState({ showSubItems: false });
+        }
+    }
+
+    render() {
+        const primaryAwardList = (<div className="primaryAwardTypeOption">
```

Comment:
Let's refactor this so we don't have a large chunk of HTML outside of the `render()` function. If you need to put this much HTML in, you may want to consider creating this as its own component.

```
let primaryAward = <CollapsedAwardType />
if (this.state.showSubItems) {
    primaryAward = ......something to show list instead
}
```